packages/devtools_app/lib/src/screens/debugger/program_explorer.dart

```diff
@@ -289,25 +291,57 @@ class _FileExplorer extends StatelessWidget {
   final Function(VMServiceObjectNode) onItemSelected;
   final Function(VMServiceObjectNode) onItemExpanded;
 
+  double get selectedNodeOffset =>
```

Comment:
let's move this getter to the widget state instead

Reply:
Done